### PR TITLE
remove master references.

### PR DESCRIPTION
**Pull request by surfnerd**
Renamed master branch to main so moved links to main.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# From https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/supported-tags.md
+# From https://gitlab.com/nvidia/container-images/cuda/blob/main/doc/supported-tags.md
```
Comment (chriselion) [inline Dockerfile:1]: External, not renamed

Comment (surfnerd) [reply Dockerfile:1]: 👍

File: docs/dox-ml-agents.conf [hunk 2]
```diff
@@ -1294,7 +1294,7 @@ CHM_FILE               =
 HHC_LOCATION           =
 
 # The GENERATE_CHI flag controls if a separate .chi index file is generated
-# (YES) or that it should be included in the master .chm file (NO).
+# (YES) or that it should be included in the main .chm file (NO).
```
Comment (chriselion) [inline docs/dox-ml-agents.conf:1297]: Don't think it matters here (but still reads fine)

**Comment by chriselion** [comment]
Need to add the branch target for yamato triggers and github actions.

There are also some links in the pull request templates.

Try `git grep -i master` and see if anything else looks relevant

**Review by chriselion — APPROVED** [review]

**Pull request merged**